GOVERNANCE-maintainer.md

```diff
@@ -112,29 +114,42 @@ Any Maintainer may suggest a request for CNCF resources, either in the
 [mailing list](TODO: link to developer/maintainer mailing list), or during a
 meeting.  A simple majority of Maintainers approves the request.  The Maintainers
 may also choose to delegate working with the CNCF to non-Maintainer community
-members.
+members, who will then be added to the [CNCF's Maintainer List](https://github.com/cncf/foundation/blob/main/project-maintainers.csv)
+for that purpose.
 
 ## Code of Conduct
 
-<!-- This assumes that your project does not have a separate Code of Conduct
-Committee; most maintainer-run projects do not.  Remember to place a link
-to the private Maintainer mailing list or alias in the code-of-conduct file.-->
-
 [Code of Conduct](./code-of-conduct.md)
 violations by community members will be discussed and resolved
-on the [private Maintainer mailing list](TODO).  If the reported CoC violator
-is a Maintainer, the Maintainers will instead designate two Maintainers to work
-with CNCF staff in resolving the report.
+on the [private Maintainer mailing list](TODO).  If a Maintainer is directly involved
+in the report, the Maintainers will instead designate two Maintainers to work
+with the CNCF Code of Conduct Committee in resolving it.
+
+## Security Response Team
+
+The Maintainers will appoint a Security Response Team to handle security reports.
+This committee may simply consist of the Maintainer Council themselves.  If this
+responsibility is delegated, the Maintainers will appoint a team of at least two 
+contributors to handle it.  The Maintainers will review who is assigned to this
+at least once a year.
+
+The Security Response Team is responsible for handling all reports of security
+holes and breaches according to the [security policy](TODO:Link to security.md).
```

Comment:
Here is the template provided by TAG security, in case we want to link to it https://github.com/cncf/tag-security/blob/main/project-resources/templates/SECURITY.md to help them make a security.md

Reply:
Except that belongs in the documentation, not the template, so I'll add it there.